src/utils/authentication/authService.js

```diff
@@ -0,0 +1,98 @@
+import auth0 from 'auth0-js';
+import getEnvironmentVariables from '../getEnvironmentVariables';
+import {
+  ACCESS_TOKEN_KEY,
+  BEARER_TOKEN,
+  EXPIRES_AT_KEY,
+  USER_ID,
+  USER_PROFILE
+} from '../../constants/storageKeys';
+
+const env = getEnvironmentVariables();
+
+class AuthService {
+  returnUrl = window.location.origin;
+
+  webAuth = new auth0.WebAuth({
+    domain: env.auth0Domain,
+    clientID: env.auth0ClientId,
+    audience: env.auth0ApiIdentifier,
+    redirectUri: `${window.location.origin}/callback`,
+    responseType: 'token id_token',
+    scope: 'openid email profile'
+  });
+
+  login = () => {
+    this.webAuth.authorize();
+  };
+
+  signUp = () => {
+    this.webAuth.authorize();
```

Comment:
I think we need to pass `{ initialScreen: 'signUp' }` to get our customized auth0 login page to show the signup tab when navigated to.